openlogin/new-v2-user-login/index.config.ts

```diff
@@ -0,0 +1,33 @@
+import path from "path";
+import {
+  PlaywrightTestConfig,
+  PlaywrightWorkerOptions,
+} from "@playwright/test";
+import { TestArgs } from "./index.lib";
+import indexConfig from "../../index.config";
+
+const testUserEmail = `testuser${process.env.GITHUB_JOB}@tor.us`
```

Comment:
Using Github Actions job id to create collision-free unique emails.

Reply:
Please use `@openlogin.com` email.

Also, `testuser_${process.env.TEST_RUN_ID}@tor.us` may be a better name, as this test doesn't depend on Github Actions, it can run anywhere. In Github Actions, we can pass

```yml
    # ...
    env:
       TEST_RUN_ID: ${{ github.run_id }}_${{ github.run_attempt }}
```